src/Initializer/Parameters/ParameterReader.cpp

```diff
@@ -16,7 +16,7 @@ std::optional<std::string> ParameterReader::readPath(const std::string& field) {
   const auto fileName = read<std::string>(field);
   if (fileName.has_value()) {
     const auto lastPath = filesystem::path(rootPath);
-    const auto nextPath = filesystem::path(fileName.value());
+    auto nextPath = filesystem::path(fileName.value());
```

Comment:
Due to some C++ rules, it would be possible to avoid some copy in the lambda, but only _if_ that object is not const. It also seemed counter-intuitive to me, but ok. :)